src/v/raft/consensus.cc

```diff
@@ -500,8 +500,6 @@ bool consensus::should_skip_vote(bool ignore_heartbeat) {
     }
 
     skip_vote |= _vstate == vote_state::leader; // already a leader
-    skip_vote |= !_configuration_manager.get_latest().is_voter(
```

Comment:
does this change mean that `vote_stm::vote` would now allow a non-voter cast a vote?

Reply:
non-voters were always allow to cast a vote but not to request a vote. This behavior didn't change